tutorials/admin/services/services.md

```diff
@@ -1 +1,75 @@
-# services
+# Services
+
+The Marble software can be configured to provide several services to the users of your node. Services are user-facing
+tools that largely fall under the following categories:
+
+- serving and cataloging data hosted on your server
+- interactive development environment
+- remote processing tools
+
+(enable)=
+## Enable an included service
+
+You can pick and choose which services you would like to offer your users by enabling them via the `env.local` file.
```

Comment:
Let's preface this sentence with "The Birdhouse stack comes loaded with several optional services." Or something to that effect.

Reply:
Actually, you see to have something like that in the next paragraph. I'd make that as the leading paragraph of this section and then add move this sentence to the end of that paragraph.